lumicks/pylake/calibration.py

```diff
@@ -1,3 +1,218 @@
+import re
+from collections import UserDict
+
+
+def _read_from_bl_dict(bl_dict):
+    mapping = {
+        "bead_diameter": "Bead diameter (um)",
+        "rho_bead": "Bead density (Kg/m3)",
+        "rho_sample": "Fluid density (Kg/m3)",
+        "viscosity": "Viscosity (Pa*s)",
+        "temperature": "Temperature (C)",
+        "driving_frequency_guess": "Driving data frequency (Hz)",
+        "distance_to_surface": "Bead center height (um)",
+        # Fixed diode parameters (only available when diode was fixed)
+        "fixed_alpha": "Diode alpha",
+        "fixed_diode": "Diode frequency (Hz)",
+        # Only available for axial with active
+        "drag": "gamma_ex_lateral (kg/s)",
+    }
+
+    return {key_param: bl_dict.get(key_bl) for key_param, key_bl in mapping.items()}
+
+
+class ForceCalibrationItem(UserDict):
+    def _verify_full(self):
+        if (kind := self.data.get("Kind", "Unknown")) != "Full calibration":
+            raise ValueError(
+                "These parameters are only available for a full calibration. Instead, this is a "
+                f"calibration item where the following operation was applied: {kind}."
+            )
+
+    def power_spectrum_params(self):
```

Comment:
I was doubting about that to be honest.

The reason I left it as a non-property is because I was thinking about leaving the option open to allow overriding arguments directly in this call in the future already. In case people end up finding the merge operation with dictionaries (`params | other params`) confusing.

Reply:
yeah ok, it makes sense to leave it like this then